src/interactive/azext_interactive/azclishell/app.py

```diff
@@ -464,13 +502,89 @@ def example_repl(self, text, example, start_index, continue_flag):
                         start_index += 1
                         cmd += " " + answer.split()[-1] + " " +\
                                u' '.join(text.split()[start_index:start_index + 1])
+            self.completer.enable_scenario_recommender(True)
             example_cli.exit()
             del example_cli
         else:
             cmd = text
 
         return cmd, continue_flag
 
+    def handle_scenario(self, text):
+        """ parses for the scenario recommendation """
+        selected_option = text.partition(SELECT_SYMBOL['example'])[2].strip()
```

Comment:
Same as above

Reply:
Done

Reply:
Why the index is `2` instead of `1`?

Reply:
> For `:: 1`, why the index is `2` instead of `1`?

For method `partition`, it returns a tuple of 3 elements. The substring before the separator, the separator, the part after the separator. So index of the substring after the separator is always `2`.

Reply:
> For method partition, it returns a tuple of 3 elements. The substring before the separator, the separator, the part after the separator. So index of the substring after the separator is always 2.

OK, please add a comment to explain this reason.